packages/commonwealth/client/scripts/views/pages/search/helpers.tsx

```diff
@@ -213,7 +213,7 @@ type MemberResultRowProps = {
   setRoute: any;
 };
 const MemberResultRow = ({ addr, setRoute }: MemberResultRowProps) => {
-  const { community, address } = addr.addresses[0];
+  const { chain: community, address } = addr.addresses[0] as any;
```

Comment:
Do we not have a more verbose type for this?

Reply:
Not sure why but the `addr.addresses[0]` contained "community" instead of "chain" (which broke the profile link), updated type, lmk if it helps